src/gwc/src/main/java/org/geoserver/gwc/config/GWCInitializer.java

```diff
@@ -91,7 +92,7 @@ public void initialize(final GeoServer geoServer) throws Exception {
         LOGGER.info("Initializing GeoServer specific GWC configuration from "
                 + GWCConfigPersister.GWC_CONFIG_FILE);
 
-        final Version currentVersion = new Version("1.0.0");
+        final Version currentVersion = new Version("2.0.0");
```

Comment:
I don't know how the versioning is defined (I should have asked) ... but yeah since we are keeping backwards compatibility this should be a minor.